core/src/main/java/hudson/tasks/Maven.java

```diff
@@ -137,6 +137,12 @@
      */
     private GlobalSettingsProvider globalSettings;
 
+    /**
+     * Inject build variables as properties into maven process.
+     * @since TODO
+     */
+    private boolean injectBuildVariables = true;
```

Comment:
@KostyaSha yes - but restricted to buildVars as systemProperties - which I think may need a little more thightening of the name - but that's nit picking on my part!